pkgs/development/python-modules/pystardict/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, setuptools-scm
+, six
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pystardict";
+  version = "0.8";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "lig";
+    repo = pname;
```

Comment:
```suggestion
    repo = "pystardict";
```

Reply:
Updated. I have a couple other ones that use pname in the repo, should I got back and fix those?